src/lib/storage/table.cpp

```diff
@@ -71,6 +72,16 @@ Chunk& Table::get_chunk(ChunkID chunk_id) { return _chunks.at(chunk_id); }
 
 const Chunk& Table::get_chunk(ChunkID chunk_id) const { return _chunks.at(chunk_id); }
 
-void Table::compress_chunk(ChunkID chunk_id) { throw std::runtime_error("Implement Table::compress_chunk"); }
+void Table::compress_chunk(ChunkID chunk_id) {
+  auto new_chunk = Chunk();
+
```

Comment:
`auto &chunk = get_chunk(chunk_id);` Reference to chunk prevents multiple access using operator[]